app/views/static/index.html.erb

```diff
@@ -17,42 +19,40 @@
 <%#= render partial: 'shared/press_list', locals: {items: @press}%>
 <div class="tea-explainers container">
   <div class="tea-explainer">
-    <%= image_tag 'landing/1.png', class: ['right', 'tea-explainer-illustration'] %>
-    <div class="tea-explainer-copy left">
-    	<h3 class="tea-explainer-header">
-    	  You don’t know most of the people around you
-    	</h3>
+    <%= image_tag 'landing/1.png', class: ['left', 'tea-explainer-illustration'] %>
+    <div class="tea-explainer-copy right">
+    	<h2 class="tea-explainer-header">
+    	  It’s weird.
+    	</h2>
     	<p class="tea-explainer-body">
-    	  And current ways of meeting new people suck. There’s usually strings
-    	  attached, like a business deal or a date. Most conversations never make it
-    	  past happy hour talk of “what do you do?” You never get to talk about
-    	  anything real. About anything that makes us who we are. But what if there
-    	  was a different way?
+    	  Everyone at tea time is stepping a little out of their comfort zone. 
+        This makes it so much easier to actually learn something unexpected 
+        about the people around you. Because open minds are a prerequisite around here.
     	</p>
     </div>
   </div>
   <div class="tea-explainer">
-    <%= image_tag 'landing/2.png', class: ['left', 'tea-explainer-illustration'] %>
-    <div class="tea-explainer-copy right">
-    	<h3 class="tea-explainer-header">Tea times create space for good
-    	  conversations with good people
-    	</h3>
+    <%= image_tag 'landing/2.png', class: ['right', 'tea-explainer-illustration'] %>
+    <div class="tea-explainer-copy left">
```

Comment:
The directions are swapped so that when you reflow for mobile everything ends up properly aligned 
![screen shot 2015-05-05 at 11 47 40 pm](https://cloud.githubusercontent.com/assets/380810/7488594/31276096-f381-11e4-9943-388ce2da58d1.png). I would probably change this back.